### PR TITLE
intial commit

### PR DESCRIPTION
**Pull request by d-beschastnyi**
https://d-beschastnyi.github.io/layout_search-bar-airbnb/
https://d-beschastnyi.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Try to pass tests. You didn't aplly fonts. If you have any questions feel free to ask in Slack

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  font-size: 16px;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search__input:hover {
+  border: 1px solid #e1e7ed;
```
Comment (Viktor-Kost) [inline src/style.css:53]: create variables for repeated colors

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```
Comment (Viktor-Kost) [inline src/style.css:55]: Has no difference with a static condition
```suggestion

```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  margin: 0;
+  padding: 0;
```
Comment (Viktor-Kost) [inline src/style.css:17]: html does not have default margin and padding
```suggestion

```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search {
+  display: block;
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
```
Comment (Viktor-Kost) [inline src/style.css:27]: You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

File: src/style.css (hunk 1 above)
Comment (Viktor-Kost) [inline src/style.css:53]: Has no difference with a static condition
```suggestion

```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+.search__input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search__input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```
Comment (Viktor-Kost) [inline src/style.css:61]: Has no difference with a static condition
```suggestion

```

File: src/style.css (hunk 1 above)
Comment (d-beschastnyi) [reply src/style.css:53]: i removed the duplicate lines you mentioned below

this color is no longer repeated in the code

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+.search__icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__input {
+  box-sizing: border-box;
```
Comment (AlisaVlasova) [inline src/style.css:40]: move it to * selector

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  height: 70px;
```
Comment (AlisaVlasova) [inline src/style.css:41]: don`t hardcode height, use vertical paddings

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  outline: none;
```
Comment (AlisaVlasova) [inline src/style.css:57]: move it to search__input without focus

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.search__input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
```
Comment (AlisaVlasova) [inline src/style.css:66]:
```suggestion
.search__input::placeholder {
  font-size: 16px;
  font-weight: 300;
  color: #3d4e61;
}
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  display: block;
+  position: relative;
+  /* height: 70px; */
```
Comment (ivanchuk-yurii) [inline src/style.css:32]: remove comments

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
```
Comment (ivanchuk-yurii) [inline src/style.css:21]: it's default value

File: src/index.html [hunk 12]
```diff
@@ -11,6 +11,49 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+      <label class="search">
```
[the remark this answers is not in the document]
Comment (d-beschastnyi) [reply src/index.html:20]: linte shows me an error if the label is removed
Снимок экрана 2022-12-10 в 19 57 41

**Review by OlEzhA0 — APPROVED** [review]
Good Job!

**Pull request closed without merging**